docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -51,3 +51,6 @@ connector:
       JDBC query. Using a large timeout can potentially result in more detailed
       dynamic filters. However, it can also increase latency for some queries.
     - ``20s``
+  * - ``postgresql.include-system-tables``
```

Comment:
This file is `jdbc-common-configurations.fragment` which is used from several connectors. Please move to `postgresql.rst`.